docs/user/integration/aws-cloudwatch/README.md

```diff
@@ -0,0 +1,152 @@
+# Integrate Kyma with Amazon CloudWatch and AWS X-Ray
+
+## Overview
+
```

Comment:
Consider adding the overview table that the other integration guides have.
Please verify the following suggestion, I'm just guessing.

```
| Category| |
| - | - |
| Signal types | traces, logs, metrics |
| Backend type | third-party remote |
| OTLP-native | yes, but AWS distro in parallel |
```

Reply:
I added the overview table.
For the OTLP-native row, we should simply state that it is not supported. At the end, all the signals which are sent to CloudWatch and X-ray are not in OTLP format.

Reply:
I checked with @chrkl and he also agreed that it should be simply no.
But we can have a third opinion from @a-thaler

Reply:
Ah then it's fine. Andreas was the man who invented this table, so I thought of him first; but @chrkl's confirmation is fine with me too!